src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -157,13 +157,12 @@ def get_status(self, tank_index: int, service: ServiceType) -> RunningStatus:
                 break
         return RunningStatus.RUNNING if ready else RunningStatus.PENDING
 
-    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user: str = "root"):
-        # k8s doesn't let us run exec commands as a user, but we can use su
-        # because its installed in the bitcoin containers. we will need to rework
-        # this command if we decided to remove gosu from the containers
-        # TODO: change this if we remove gosu
+    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user="root"):
```

Comment:
in "images: unified alpine-based dockerfile" (https://github.com/bitcoin-dev-project/warnet/commit/4dc71ff74d4267b9c3f45530c1490702bd81c188):

```suggestion
    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user: str = "root"):
```

Reply:
taken